### ISSUE TITLE
Update examples to use PlatformSupport.makeEventLoopGroup?

### ISSUE THREAD

**Issue by Jerry-Carter** [grpc/grpc-swift#1324]
### Describe the bug

According to the [Using TLS](https://github.com/grpc/grpc-swift/blob/main/docs/tls.md) guide, the use of 

`PlatformSupport.makeEventLoopGroup(loopCount: System.coreCount)`

is preferred over calling MultiThreadedEventLoopGroup for TLS.  Presumably this is generally true and the Examples should be updated accordingly.

**Comment by glbrntt** [comment]
Yes, ideally they would make use of `PlatformSupport.makeEventLoopGroup`
